Podfile

```diff
@@ -16,6 +16,7 @@ def ios_pods
     pod 'PinpointKit', :git => 'https://github.com/Lickability/PinpointKit', :commit => 'dd5731c'
     pod 'DZNEmptyDataSet', '1.8.1'
     pod 'Hero'
+    pod 'SimpleRoundedButton'
```

Comment:
But for now we don't use any of that right? There is no loading indicator on this button yet. And I don't know how 'iOS-like' this style is, I'll have to get a better look on those new iOS 11 designs